game/tap.ksy

```diff
@@ -0,0 +1,49 @@
+meta:
+  id: tap
```

Comment:
Sounds good. Will do.

Reply:
I'm actually wondering is `game` is a good primary category as well. ZX Spectrum is not just a gaming appliance — and this looks like it's a general purpose tape emulator image format. May be `filesystem` will be a better candidate?

Reply:
Right, `game` may not be the best category. `filesystem` isn't entirely accurate either since the TAP format is a wrapper around the actual data found on the tape, but on the other hand it's a *very* thin wrapper (it just adds length bytes) and I can't think of any better category right now.

Reply:
Everyone happy to move to `filesystem` then?